blocks/feynman/__link/feynman__link.css

```diff
@@ -0,0 +1,17 @@
+.feynman__link {
+    position: absolute;
+    top: 50%;
+    margin-right: 48px;
+    align-self: flex-end;
+    line-height: 51px;
+    font-size: 36px;
+    font-weight: normal;
+    text-decoration: none;
+    color: #2f80ed;
+    transition: all 0.5s ease-in-out;
+}
+
+.feynman__link:hover {
+    opacity: 0.5;
+    transition: all 0.5s ease-in-out;
+}
```

Comment:
The `transition: all 0.5s ease-in-out` property is not needed for the pseudo-class hover since it is already defined for the element.